lib/stages/VisSharedMemWriter.cpp

```diff
@@ -55,7 +55,9 @@ VisSharedMemWriter::VisSharedMemWriter(Config& config, const std::string& unique
                                        bufferContainer& buffer_container) :
     Stage(config, unique_name, buffer_container, std::bind(&VisSharedMemWriter::main_thread, this)),
     dropped_frame_counter(Metrics::instance().add_counter(
-        "kotekan_vissharedmemwriter_dropped_frame_total", unique_name, {"freq_id", "reason"})) {
+        "kotekan_vissharedmemwriter_dropped_frame_total", unique_name, {"freq_id", "reason"})),
+    access_record_wait_time_seconds(Metrics::instance().add_gauge(
+        "kotekan_access_record_wait_time_seconds", unique_name, {"shm_name"})) {
```

Comment:
I think this one should also be prefixed with `kotekan_vissharedmemwriter_` (see https://github.com/kotekan/kotekan/blob/9be2a6f0a0bc237d6e29f4e5cdb93d4869c5edf7/lib/core/prometheusMetrics.hpp#L305)